client/app/dedicatedCloud/datacenter/drp/dedicatedCloud-datacenter-drp.routes.js

```diff
@@ -0,0 +1,23 @@
+angular.module('App').config(($stateProvider) => {
+  $stateProvider.state('app.dedicatedClouds.datacenter.drp', {
+    url: '/drp',
+    views: {
+      'pccDatacenterView@app.dedicatedClouds.datacenter': {
+        component: 'dedicatedCloudDatacenterDrp',
+      },
+    },
+    translations: ['../../../ip'],
```

Comment:
Maybe you could have your own translations, so the whole component would be easier to move around